Missionframework/modules/02_build/fnc/fn_build_initModule.sqf

```diff
@@ -1,19 +1,28 @@
 /*
-    KP LIBERATION MODULE INITIALIZATION
+    KPLIB_fnc_build_initModule
 
-    File: initModule.sqf
+    File: fn_build_initModule.sqf
     Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
-    Date: 2018-07-01
-    Last Update: 2018-07-01
+    Date: 2017-08-31
+    Last Update: 2018-08-31
     License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
 
     Description:
     This module provides building functionalities
 
     Dependencies:
         * 01_core
+
+    Returns:
+    BOOL
 */
 
+if (isServer) then {diag_log format ["[KP LIBERATION] [%1] [BUILD] Module initializing...", diag_tickTime];};
+
 call compile preprocessFileLineNumbers "modules\02_build\globals.sqf";
 
 call KPLIB_fnc_build_setupPlayerActions;
+
+if (isServer) then {diag_log format ["[KP LIBERATION] [%1] [BUILD] Module initialized", diag_tickTime];};
```

Comment:
Are the `diag_log`s executions only on server intentional? The modules initialization usually is ran on both client and server.

Reply:
My thought was:
Mostly you look at the server log for errors instead of the logs of all clients. So I just wanted to "spam" the server log. But I could just remove the if and log it everywhere.

Reply:
This reasoning is fine for me.